packages/nimbus/src/components/form-field/form-field.mdx

```diff
@@ -197,53 +202,76 @@ const App = () => {
 }
 ```
 
-
 ## Guidelines
 
 Design guidelines offer a set of standardized rules and recommendations that
 ensure consistency, usability, and accessibility across all design elements and
 interactions.
 
-Related guidelines see [labels](#) and [text-fields](#).
+Related guidelines [text-fields](#).
 
 ### Best practices
 
 - **Clarity and simplicity:**
-  - Minimize the number of fields. Only ask for essential information.  
+  - Minimize the number of fields. Only ask for essential information.
   - Use clear, concise labels that accurately describe the required input.
   - Choose what position (top or left) the labels will be and make it consistent
     in the area to help with scanning.
 - **Labels:**
   - Place labels above or to the left of input fields.
-  - Keep labels short and descriptive.  
-  - Avoid using placeholders as labels.  
+  - Keep labels short and descriptive.
+  - Avoid using placeholders as labels.
 - **Error handling:**
-  - Display error messages clearly and prominently.  
+  - Display error messages clearly and prominently.
   - Use specific and helpful error messages and provide suggestions for
     correcting errors.
 
+#### Labels
```

Comment:
Wasn't sure of the best place to put this, so I guessed

Reply:
I think I would vote for putting it here too.